### PR TITLE
fix harbor error when parse json with unicode;change pritn to print f…

### PR DESCRIPTION
**Pull request by xidianwlc**
修复simplejson依赖错误，修复pritn语法错误，修正编码规范支持pep8

### CODE REVIEW

File: harborclient/harborclient.py [hunk 1]
```diff
@@ -1,13 +1,16 @@
 #!/usr/bin/env python
 
-import requests
-import simplejson
+# for inner
+import json
 import logging
+# for thirdparty
```
Comment (tobegit3hub) [inline harborclient/harborclient.py:6]: I'm not sure if we need this. The third party dependencies will be listed in `requirements.txt`.

File: harborclient/harborclient.py [hunk 2]
```diff
@@ -52,7 +55,7 @@ def get_project_id_from_name(self, project_name):
                     project_id, project_name))
             return project_id
         else:
-            pritn("Fail to get project id from project name", project_name)
+            print("Fail to get project id from project name", project_name)
```
Comment (tobegit3hub) [inline harborclient/harborclient.py:58]: Nice catch. This should be changed to use `logging.error()` and I missed it.

**Comment by tobegit3hub** [comment]
Looks great and thanks for your help.

Do you mind changing for my comment? Or I will merge this first.

File: harborclient/harborclient.py (hunk 1 above)
Comment (xidianwlc) [inline harborclient/harborclient.py:6]: if include simplejson some unicode parse error will happen，and the protocol of json is the same with simplejson. The best way is replace simplejson to json

Comment (tobegit3hub) [inline harborclient/harborclient.py:6]: Sorry about my statement. I'm confused about the comment and thanks for your explaination.

**Pull request merged**